PicaSWB.js

```diff
@@ -104,6 +123,16 @@ function doExport() {
 		} else {
 			titleStatement += item.title.replace(/\s*:\s*/,'$d');
 		}
+		//Sortierzeichen hinzufügen, vgl. https://github.com/UB-Mannheim/zotkat/files/137992/ARTIKEL.pdf
```

Comment:
ja, eine "OR" Bedingung ist sicherlich gut, falls ein Bearbeiter vergisst das Sprachfeld vor dem Kopiervorgang in WinIBW einzutragen.